MLAPI/Spawning/SpawnManager.cs

```diff
@@ -304,8 +304,7 @@ internal static NetworkedObject CreateLocalNetworkedObject(bool softCreate, ulon
                 // SoftSync them by mapping
                 if (!pendingSoftSyncObjects.ContainsKey(instanceId))
                 {
-                    // TODO: Fix this message
-                    if (NetworkLog.CurrentLogLevel <= LogLevel.Error) NetworkLog.LogError("Cannot find pending soft sync object. Is the projects the same?");
+                    if (NetworkLog.CurrentLogLevel <= LogLevel.Error) NetworkLog.LogError(String.Format("Cannot find pending SoftSync object [InstanceID={0}]. Are the projects the same?", instanceId));
```

Comment:
```suggestion
                    if (NetworkLog.CurrentLogLevel <= LogLevel.Error) NetworkLog.LogError($"Cannot find pending SoftSync object [InstanceID={ instanceId }]. Are the projects the same?");
```

Reply:
Oh awesome! I’m a bit of a C# noob so I had no idea that even existed. Thanks!